### PR TITLE
Vim

### PR DESCRIPTION
**Pull request by DerTim1**
###### Motivation for this change
Update vim 8.0.0005 -> 8.0.0075

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] Linux (Nix on Ubuntu 16.04)
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -4556,6 +4556,27 @@ in
...
+  #  };
+  #
+  #  elixir = callPackage ../development/interpreters/elixir { };
+  #
+  #  nodejs = callPackage ../development/web/nodejs/v6.nix { 
+  #    libtool = darwin.cctools;
+  #  };
+  #};
+
+  ec-compiler = with pkgs; buildEnv {
```
Comment (8573) [inline pkgs/top-level/all-packages.nix]: Is this related to Vim?

**Comment by DerTim1** [comment]
Sorry messed up something with Git. I'll fix this an create a new PR. This PR can be deleted.
